ui/app/components/selectable-card-container-graph.js

```diff
@@ -0,0 +1,42 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+/**
+ * @module MetricsSelectableCardContainerGraph
+ * MetricsSelectableCardContainerGraph components are used to...
+ *
+ * @example
+ * ```js
+ * <MetricsSelectableCardContainerGraph @requiredParam={requiredParam} @optionalParam={optionalParam} @param1={{param1}}/>
+ * ```
+ * @param {object} requiredParam - requiredParam is...
```

Comment:
I _think_ we also want to fill these in with actual params -- in this case, `@counters` (the shape of counters would be extra useful too!)

Reply:
yep, that's correct! we use these comments to automatically generate the 'notes' pages in Storybook like [this one](https://vault-storybook.netlify.com/?path=/info/alerts-alertpopup--alertpopup). later on when you add this component to storybook, you can do so by following the instructions [here](https://github.com/hashicorp/vault/tree/master/ui#adding-a-new-story).

Reply:
automatic notes 😍 love it

Reply:
Good to know.  I wasn't sure if I should keep or remove this generated comment.  But I love that it will help with Storybook.  I'll amend.